_notebooks/textbook.py

```diff
@@ -130,3 +144,22 @@ def clean_raw_text(self, disable_pysbd=False):
             if not disable_pysbd:
                 clean_text = "\n".join(seg.segment(clean_text))
             chapter.clean_text = clean_text
+    
```

Comment:
Should we rename the function `clean_raw_text` to something more descriptive like `improve_sentence_boundaries`?